src/main/java/com/jalasoft/webservice/model/AudioConvert.java

```diff
@@ -55,9 +61,11 @@ public Response convert(Criteria criteria) throws IOException {
             String [] filePaths = new String[5];
             filePaths[0]=audiocri.getTarget();
             zipFiles.zipFiles(filePaths);
+            logger.info("Audio conversion succesfully - Method: " + new Object() {}.getClass().getEnclosingMethod().getName());
```

Comment:
review this line do not overload the 100 character

Reply:
Done